static/js/gear_menu.js

```diff
@@ -41,6 +41,23 @@ exports.initialize = function () {
         }
     });
 
+    $('#contact_support').on('click', function (e) {
+        new ClipboardJS('#contact_support', {
+            text: function () {
+                var data = page_params.zulip_administrator;
+                return data;
+            },
+        });
+        e.stopPropagation();
+        $(this).html('<i class="fa fa-address-book" aria-hidden="true"></i> Address Copied');
+        $(this).css("color", "#2d7c6f");
+        $(this).css("background-image", "linear-gradient(to bottom, #fff, #fff");
```

Comment:
Not sure if we usually do inline-css. I'd think we just create a class and then just apply the class to the HTML.